apps/docs/src/app/platform/component-docs/platform-file-uploader/platform-file-uploader-examples/platform-file-uploader-compact-example.component.html

```diff
@@ -2,15 +2,15 @@
     <fdp-form-field
         #ffl1
         label="Default Input Field"
-        [id]="'input1'"
+        id="input1"
         name="reactiveFormInput"
         zone="“zLeft”"
```

Comment:
the  extra quotes in this looks odd `zone="“zLeft”"`. Please check at other places too